src/frontend.cpp

```diff
@@ -0,0 +1,16 @@
+#include "frontend.h"
+
+void Frontend::GenerateStringStream() {
+	stream.str(drv->out);
+}
+
+void Frontend::WriteFile() {
+	GenerateStringStream();
+	if (outputFileName.empty()) {
+		outputFileName = defaultFilename;
+	}
+	fileStream.open(outputFileName);
+	fileStream << stream.rdbuf();
+	fileStream.close();
+	std::cout << "file generated" << std::endl;
```

Comment:
Store bogstaver til at starte med? Ideelt burde man måske skrive noget hen af "Output written to <filename>".